dist/index.html

```diff
@@ -1,42 +1,41 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link
-      rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
-    />
-    <link
-      href="https://fonts.googleapis.com/css?family=Quintessential:regular"
-      rel="stylesheet"
-    />
-    <title>Todo List</title>
-  <script defer src="runtime.bundle.js"></script><script defer src="index.bundle.js"></script></head>
-  <body>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+<script defer src="runtime.bundle.js"></script><script defer src="index.bundle.js"></script></head>
+<body>
     <main>
-      <section class="container">
-        <div class="header">
-          <h1>Paul's Todo</h1>
-          <div class="refresh">
-            <i class="fa fa-refresh" aria-hidden="true"></i>
-          </div>
-        </div>
-        <form class="form-container">
-          <input
-            id="input"
-            type="text"
-            placeholder="Add to your list..."
-            class="add-to-your-list"
-            name="todo"
-          />
-        </form>
-        <ul class="todo-items"></ul>
-        <div class="clear-completed">
-          <span class="complete">Clear all completed</span>
-        </div>
-      </section>
+        <section class="container">
+            <h1>Leaderboard</h1>
+            <div class="row score-container">
+                <div class="col-6 inner-left-container">
+                    <div class="inner-left-header">
+                        <h3>Recent scores</h3>
+                        <button type="button">Refresh</button>
+                    </div>
+                    <div class="scores">
+                        <p>Paul: <span>100</span></p>
+                        <p>Paul: <span>70</span></p>
+                        <p>Paul: <span>125</span></p>
+                        <p>Paul: <span>150</span></p>
+                        <p>Paul: <span>120</span></p>
+                    </div>
+                </div>
+                <div class="col-6 inner-right-container">
+                        <div class="inner-right-header">
+                            <h3>Add your score</h3>
+                        </div>
+                        <form>
+                            <input id="name" type="text" placeholder="Your name"/>
+                            <input id="score" type="text" placeholder="Your score"/>
+                        </form>
+                        <button id="submit-btn" type="submit">Submit</button>
+                </div>
+            </div>
+        </section>
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly ensure your design matches the wireframe below: 

*wireframe*
![matchhh](https://user-images.githubusercontent.com/59397159/173662651-a0c2d752-b6e2-466b-817d-2ec263d4ee6b.png)

*your design*
![match](https://user-images.githubusercontent.com/59397159/173662654-0d0f7478-e401-400e-bbbf-b715c3bbff18.png)